packages/assets/scss/03-organisms/_illustrated-header.scss

```
@@ -105,6 +98,30 @@ $bp-illustrated-header-image-width: 360px;
       width: 100%;
     }
   }
+
+  &--no-image {
+    .ma__illustrated-header__image {
+      display: none;
+    }
+
+    .ma__illustrated-header__content {
+      @media ($bp-page-width-min) {
+        width: 935px;

```

```suggestion
        width: 935px;

```
How is this width determined? Are we using this title width in other headers?

---

The width comes from the mockup, the new topic page header is somewhat similar: https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-22640_topic-page-redesign-fe/?p=organisms-page-banner-as-white
Although in that case is 1050px width.

---

ok, thanks for noting that. I will approve this for now, but for the topic header, let's reconvene with Marissa, I think should come up something more consistent for page headers across the board. 

For references, the width on desktop for the page content is 738px, and we have these numbers in the system:

$l-max-content-width:           820px;
$l-max-content-wrapper:         1240px;
// Semi Global breakpoint media variables
// switch the header from full width to hamburger
$bp-header-toggle-min:      "min-width: 841px" !default;
$bp-header-toggle-max:      "max-width: 840px" !default;
$bp-xs:   480px !default;
$bp-s:    620px !default;
$bp-m:    780px !default;
$bp-l:    910px !default;
$bp-l-ex: 1050px !default;
$bp-xl:   1200px !default;